### PR TITLE
small doc updates

### PR DESCRIPTION
**Pull request by kazimir-malevich**
You can fetch upstream in the GH GUI now. I think these are the only changes that needed to be made after upstream fetch. Not sure about line 97 onward. I'm using Linux are you on Windows (or someone on your team?).

- Closes https://github.com/tag1consulting/goose/pull/463

### CODE REVIEW

File: src/docs/goose-book/src/config/defaults.md [hunk 1]
```diff
@@ -81,8 +81,8 @@ For example, without any run-time options the following load test would automati
             .register_transaction(transaction!(loadtest_index))
         )
         .set_default(GooseDefault::Host, "local.dev")?
-        .set_default(GooseDefault::RequestsFile, "goose-requests.log")?
-        .set_default(GooseDefault::DebugFile, "goose-debug.log")?
+        .set_default(GooseDefault::RequestsLog, "goose-requests.log")?
```
Comment (jeremyandrews) [inline src/docs/goose-book/src/config/defaults.md:84]: This is `RequestLog` not `RequestsLog` (drop the `s`):
https://docs.rs/goose/*/goose/config/enum.GooseDefault.html#variant.RequestLog

**Review by jeremyandrews — CHANGES_REQUESTED** [review]
It would have been better to rebase your earlier PR (https://github.com/tag1consulting/goose/pull/463) -- in any case, please also update the incorrect documentation on that same page as you did in your earlier PR. (`GooseDefault::RequestsLog` where it appears earlier on the page should also be `GooseDefault::RequestLog`)

**Comment by kazimir-malevich** [comment]
I am sorry, I thought I would have messed things up with rebasing.

**Review by jeremyandrews — APPROVED** [review]
Thanks for the documentation cleanup! No real problem with the duplicate PR -- if you want to try rebasing in the future I can help you with that, it's a very useful/helpful git skill to master.

**Comment by kazimir-malevich** [comment]
I think my workflow should be:

1. **fetch upstream**, 
2. Create **feature branch**, make changes, **commit**
3. Then **rebase**, **feature branch**  onto "*main**

I  am having been working on a team of one, maybe two for over a decade.

**Comment by jeremyandrews** [comment]
Correct. But you will generally only need to rebase if another PR lands in the time between when you fetched your upstream and created your feature branch. And if the merged changes are in a different area of the code there's usually nothing or very little to manually resolve (most commonly CHANGELOG.txt).

**Comment by jeremyandrews** [comment]
Some useful info here on how to do it here: https://www.w3docs.com/snippets/git/how-to-rebase-git-branch.html

**Pull request merged**